apps/condo/domains/marketplace/components/Invoice/CreateInvoiceForm.tsx

```diff
@@ -32,36 +29,21 @@ export const CreateInvoiceForm: React.FC = () => {
         },
     })
 
-    const [registerMultiPayment] = useMutation(REGISTER_MULTI_PAYMENT_MUTATION)
+    const getPaymentLink = useInvoicePaymentLink()
 
     const handleCreateInvoice = useCallback(async (values) => {
         const payload = Invoice.formValuesProcessor({ ...values, context: invoiceContext.id }, invoiceContext)
         const createdInvoice = await createInvoiceAction(payload)
 
         const { status } = values
         if (status === INVOICE_STATUS_PUBLISHED) {
-            await runMutation({
-                mutation: registerMultiPayment,
-                variables: {
-                    data: {
-                        dv: 1,
-                        sender: getClientSideSenderInfo(),
-                        invoices: [{ id: createdInvoice.id }],
-                    },
-                },
-                OnCompletedMsg: (data) => {
-                    const { data: { result } } = data
-                    return getPaymentLinkNotification({ intl, number: createdInvoice.number, url: result.webViewUrl })
-                },
-                onError: (err) => {
-                    console.log(err)
-                },
-                intl,
-            })
+            const paymentLink = await getPaymentLink(createdInvoice.id)
+
+            notification.success(getPaymentLinkNotification({ intl, number: createdInvoice.number, url: paymentLink }))
```

Comment:
Payment link can be undefined here? Any checks of that case here?

Reply:
Added check in getPaymentLink